pkg/packer/manifests/gcp/packer.pkr.hcl

```diff
@@ -80,6 +80,11 @@ variable "project_id" {
   default = ""
 }
 
+variable "image_storage_locations" {
+  type    = string
+  default = ""
```

Comment:
Is this default equivalent to the previous behavior?

Reply:
we were not setting `image_storage_locations` before so it was setting to default nearby location. 
```
image_storage_locations ([]string) - Storage location, either regional or multi-regional, where snapshot content is to be stored and only accepts 1 value. Always defaults to a nearby regional or multi-regional location.
```
The default value "" will be set to [""] for image_storage_locations
[HCL documentation.](https://developer.hashicorp.com/packer/docs/templates/hcl_templates/functions/string/split)
```
 split(",", "")
[
  "",
]
```
Let me test how packer plugin behaves in that case. AFIK it tries to set it to `global` internally which fails for our infra as expected. 

Running a test locally now. I will update in 30 mins.